### ISSUE TITLE
Error after installing new version of terra

### ISSUE THREAD

**Issue by frankcarey** [terra-ops/terra-cli#97]
I got an error:

There is an error with your configuration: The child node "domains" at path "project.apps.drupal7.environments.local" must be configured.

I had to create a new item `domains: [ ]` in ~/.terra/terra yaml file to get the `terra status` to work.

Final result looks like this:

``` Yaml
    drupal7:
        name: drupal7
        description: ''
        repo: 'git@github.com:terra-ops/example-drupal.git'
        host: 192.168.10.10
        environments:
            local:
                name: local
                path: /Users/fcarey/Apps/drupal7/local
                document_root: ''
                url: ''
                version: master
                domains: [ ]
```

**Comment by jlyon** [comment]
I'm getting this too.

**Comment by jonpugh** [comment]
Thanks guys,
I believe I fixed this with cd6d8075d35aec5947b95d1083116446e93bbb35

Pull and test for me, and close if it works?

**Comment by jonpugh** [comment]
Just installed from scratch, didn't get the error, so closing!
